src/protean/core/application_service.py

```diff
@@ -0,0 +1,10 @@
+class BaseApplicationService:
+    """Base ApplicationService class that all other Application services should inherit from.
+    This is a placeholder class for now. Methods that are implemented
+    in concreate Application Service classes are inspired from Application concepts,
+    and typically use more than one aggregate to accomplish a task"""
```

Comment:
`and typically use more than one aggregate, spread across more than one context to accomplish a task`
@subhashb:  What are your thoughts?

Reply:
Good point. Application services are still supposed to be within a bounded context. But I don't have enough empirical evidence to support this fact. If we were to imagine that each bounded context is a standalone microservice, then a single API call will not be able to talk to multiple Microservices in a straight-forward manner.

One exception is using a reactive/messaging mechanism to retrieve data from multiple microservices, but that is an exception and is associated with a specific kind of implementation.

I have now changed the entire comment to reflect functionality better.